Makefile

```diff
@@ -64,6 +64,17 @@ ifdef V8_SRCDIR
 override CPPFLAGS += -I$(V8_SRCDIR) -I$(V8_SRCDIR)/include
 endif
 
+ifeq ($(OS),Windows_NT)
```

Comment:
Not sure why you want to silently skip it.  What would be here in case of mingw?  I know some people build plv8 in it.